torch/distributed/_tensor/ops/embedding_ops.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
+# implement matrix related ops for distributed tensor
+
+import torch
+
+from torch.distributed._tensor.api import DTensorSpec, Replicate, Shard
+from torch.distributed._tensor.op_schema import OpSchema, OutputSharding
+from torch.distributed._tensor.ops.utils import register_prop_rule
+
+aten = torch.ops.aten
+
+
+@register_prop_rule(aten.embedding.default)
```

Comment:
I think we should also enable the backward of embedding call, maybe we can do it in a follow up PR.